frontend/src/poc-code/testK8s/ws/ws-utils.ts

```diff
@@ -0,0 +1,56 @@
+const getToken = (): string | null => {
```

Comment:
No need to use cookie token, we can directly use `insights.chrome.auth.getToken` or more elegant way would be to use `auth` from `useChrome` hook.

Reply:
Not going to change this. Sorry. If you think this is a problem with merging, then we are going to wait until the SDK is ready (not ideal imo -- but also not that far away). There needs to be new configurations at the App level to support token passing and that comes with my SDK PR - https://github.com/openshift/dynamic-plugin-sdk/pull/38